src/ol/interaction/Draw.js

```diff
@@ -393,6 +413,46 @@ class Draw extends PointerInteraction {
           } else {
             geometry = new Constructor(coordinates);
           }
+          const stride = getStrideForLayout(this.geometryLayout_);
```

Comment:
I'm wondering if this block could be simplified. Maybe a better approach would be to work with the specified coordinate layout's stride throughout the draw process, i.e. starting with the `sketchCoord`?

Reply:
Thanks for reviewing. I've updated the PR.